### PR TITLE
node: (cleanup) remove injectC in favor of using msgC directly

### PR DESCRIPTION
**Pull request by tbjump**

### CODE REVIEW

**Comment by tbjump** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3097** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3097" target="_blank">Graphite</a> 
    * **PR #3096** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3096" target="_blank">Graphite</a> 
      * **PR #3024** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3024" target="_blank">Graphite</a> 
        * **PR #3067** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3067" target="_blank">Graphite</a> 
          * **PR #3134** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3134" target="_blank">Graphite</a> 
            * **PR #3127** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3127" target="_blank">Graphite</a> 
              * **PR #3128** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3128" target="_blank">Graphite</a> 
                * **PR #3129** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3129" target="_blank">Graphite</a> 
                  * **PR #3130** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3130" target="_blank">Graphite</a> 
                    * **PR #3136** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3136" target="_blank">Graphite</a> 
                      * **PR #3135** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3135" target="_blank">Graphite</a> 
                        * **PR #3139** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3139" target="_blank">Graphite</a>  👈
                          * **PR #3140** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3140" target="_blank">Graphite</a> 
                        * **PR #3137** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3137" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3139?utm_source=stack-comment).

**Comment by tbjump** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3097** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3097" target="_blank">Graphite</a> 
    * **PR #3096** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3096" target="_blank">Graphite</a> 
      * **PR #3024** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3024" target="_blank">Graphite</a> 
        * **PR #3067** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3067" target="_blank">Graphite</a> 
          * **PR #3134** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3134" target="_blank">Graphite</a> 
            * **PR #3127** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3127" target="_blank">Graphite</a> 
              * **PR #3128** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3128" target="_blank">Graphite</a> 
                * **PR #3129** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3129" target="_blank">Graphite</a> 
                  * **PR #3130** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3130" target="_blank">Graphite</a> 
                    * **PR #3136** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3136" target="_blank">Graphite</a> 
                      * **PR #3135** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3135" target="_blank">Graphite</a> 
                        * **PR #3139** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3139" target="_blank">Graphite</a>  👈
                          * **PR #3140** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3140" target="_blank">Graphite</a> 
                        * **PR #3137** <a href="https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3137" target="_blank">Graphite</a> 

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/wormhole-foundation/wormhole/3139?utm_source=stack-comment).

File: node/pkg/adminrpc/adminserver.go [hunk 1]
```diff
@@ -33,10 +35,18 @@ import (
...
 )
 
+var (
+	vaaInjectionsTotal = promauto.NewCounter(
+		prometheus.CounterOpts{
+			Name: "wormhole_vaa_injections_total",
+			Help: "Total number of injected VAA queued for broadcast",
+		})
+)
+
```
Comment (SEJeff) [inline node/pkg/adminrpc/adminserver.go:45]: It might be cool to add labels for this for the chain these are injected to.

Comment (tbjump) [reply node/pkg/adminrpc/adminserver.go:45]: feel free :)

**Pull request closed without merging**